README.md

```diff
@@ -18,51 +18,6 @@ A category filtering allowing users to choose
 there adventure based on their needs. 
 A register user account for anyone around the glob
 and many other interactive features.
- 
```

Comment:
Having a package json and a requirement. txt is enough. It seems redundant to have a dependancy description given that it contains the same content.